src/wp-admin/customize.php

```diff
@@ -261,7 +261,7 @@
 							$class .= ' active';
 						}
 						?>
-						<button type="button" class="<?php echo esc_attr( $class ); ?>" aria-pressed="<?php echo esc_attr( $active ); ?>" data-device="<?php echo esc_attr( $device ); ?>">
+						<button type="button" class="<?php echo esc_attr( $class ); ?>" aria-pressed="<?php echo esc_attr( $active ? 'true' : 'false' ); ?>" data-device="<?php echo esc_attr( $device ); ?>">
```

Comment:
If the values are hard-coded strings, `'true'`/`'false'`, the call to `esc_attr()` can be removed.